test-e2e/devServer.ts

```diff
@@ -0,0 +1,43 @@
+process.title = "ajs-test-e2e-dev-server"  // set a process title so that we can call pkill to stop the server using the title
+
+import * as express from 'express';
+import * as path from 'path';
+import * as config from './config'
+import fetch from 'node-fetch'
+import * as fs from 'fs'
```

Comment:
I think we want `prettier` to hit this file as well

Reply:
Done